core/src/it/scala/org/ensime/fixture/EnsimeConfigFixture.scala

```diff
@@ -29,13 +29,18 @@ trait EnsimeConfigFixture {
   def withEnsimeConfig(testCode: EnsimeConfig => Any): Any
 
   // convenience method
-  def main(lang: String)(implicit config: EnsimeConfig): File =
-    config.projects.head.sources.filter { dir =>
+
+  def mains(lang: String)(implicit config: EnsimeConfig): List[File] =
```

Comment:
the convenience part here was in returning a single dir from a project definition where we only expected one `src/main/scala` directory. Has that changed?